### DISCUSSION TITLE
Vercel: internal_server_error: An unexpected internal error occurred

### DISCUSSION THREAD

**Discussion by kavehsaket** [vercel/community#2369] [Bug Report]
I tried to deploy next.js 13.4.1 I get the following error.
I can build my application locally without any issues.

<img width="841" alt="Screenshot 2023-05-09 at 10 45 48 am" src="https://user-images.githubusercontent.com/5524074/236966250-c34210d0-e4b2-417a-97f9-144ba6e31f20.png">

**Comment by kavehsaket** [comment]
After re-deploying 10th time fixed. I didn't change anything in my code. 😳

**Reply by amyegan** [reply]
😱 That's never a good feeling! It looks like there was a brief API outage from one of our upstream providers around the time that this happened. I believe that was the cause of the build failures. Things appear to be working normally now, but please let me know if you see this error again 🙏
